playwright.config.ts

```diff
@@ -4,22 +4,24 @@ import { defineConfig, devices } from "@playwright/test";
  * Read environment variables from file.
  * https://github.com/motdotla/dotenv
  */
+import "dotenv/config";
 
 const baseURL = process.env.CI_TEST_DEPLOYMENT_BASE_URL;
 
 /**
  * See https://playwright.dev/docs/test-configuration.
  */
 export default defineConfig({
-  testDir: "./e2e",
+  testDir: "./",
+  testMatch: "**/*.e2e-spec.ts",
   /* Run tests in files in parallel */
   fullyParallel: !process.env.CI,
   /* Fail the build on CI if you accidentally left test.only in the source code. */
   forbidOnly: !!process.env.CI,
   /* Retry on CI only */
   retries: process.env.CI ? 2 : 0,
   /* Opt out of parallel tests on CI. */
-  workers: process.env.CI ? 3 : undefined,
+  workers: process.env.CI ? 3 : 1,
```

Comment:
Multiple workers locally can run into issues when targeting the dev-server with hot module reloading and stuff.